packages/atomic/.storybook/default-story-shared.tsx

```diff
@@ -57,10 +78,10 @@ export function renderShadowPartsToStyleString(
       ).filter((rule) => rule != '');
 
       const rulesFormattedByLine = `\t${rulesForPartWithoutEmptyLines.join(
-        '\n\t'
+        ''
       )}`;
 
-      return `\n${componentTag}::part(${shadowPartName}) {\n${rulesFormattedByLine}\n}`;
+      return `\n${componentTag}::part(${shadowPartName}) {${rulesFormattedByLine}}`;
```

Comment:
This is an example of the type of code I want to be able to remove/not worry about, where I had to painfully try to align stuff manually.